src/components/Header/Header.astro

```diff
@@ -56,31 +53,41 @@ const t = useTranslations(Astro);
 				</svg>
 			</a>
 		</div>
-		<div style="flex-grow: 1;"></div>
 		{lang && <LanguageSelect lang={lang} client:idle />}
-		<div class="search-item"><Search {lang} client:idle labels={docsearchStrings} /></div>
+		<Search {lang} client:idle labels={docsearchStrings} />
+		<SidebarToggle client:idle />
 	</nav>
 </header>
 
 <style>
 	header {
 		z-index: 11;
 		height: var(--theme-navbar-height);
-		width: 100%;
+		padding: 2rem 0;
 		background-color: var(--theme-navbar-bg);
 		display: flex;
 		align-items: center;
-		justify-content: center;
+		justify-content: left;
 		overflow: hidden;
 		position: sticky;
 		top: 0;
 	}
 
+	.nav-wrapper {
+		display: flex;
+		align-items: center;
+		justify-content: center;
+		gap: 0.75rem;
+		width: 100%;
+		max-width: calc(100vw - 1.1rem);
+		padding-left: 1.5rem;
+		padding-right: 1.5rem;
+	}
```

Comment:
Nice try @hippotastic! These styles were introduced in this PR so we can also fix them here ;-)

```suggestion
		padding-left: 1rem;
		padding-right: 1.5rem;
	}
	
	@media (min-width: 50em) {
		padding-left: 1.5rem;
	}
```

Reply:
You can't compare them like this, as I moved the hamburger menu to the right, the visual paddings have remained the same. :) I've checked it side by side between old and new site.